BunqSdk/Context/ApiContext.cs

```diff
@@ -23,33 +23,33 @@ public class ApiContext
         /// <summary>
         /// Error constants.
         /// </summary>
-        private const string ERROR_COULD_NOT_SAVE_API_CONTEXT = "Could not save the API context.";
-        private const string ERROR_COULD_NOT_RESTORE_API_CONTEXT = "Could not restore the API context.";
+        private const string ErrorCouldNotSaveApiContext = "Could not save the API context.";
+        private const string ErrorCouldNotRestoreApiContext = "Could not restore the API context.";
 
         /// <summary>
         /// Measure of any time unit when none of it is needed.
         /// </summary>
-        private const int TIME_UNIT_COUNT_NONE = 0;
+        private const int TimeUnitCountNone = 0;
 
         /// <summary>
         /// Minimum time to session expiry not requiring session reset.
         /// </summary>
-        private const int TIME_TO_SESSION_EXPIRY_MINIMUM_SECONDS = 30;
+        private const int TimeToSessionExpiryMinimumSeconds = 30;
```

Comment:
TimeToSessionExpiryMinimumSecond (hmm, looks weird, @OGKevin 👀 )

Reply:
🤷‍♂️ your suggestion is fine, 🙅‍♂️ plurals.

Reply:
@OGKevin @sandervdo TimeToSessionExpiryAllMinimumSecond 🙈 This renaming changes will bring a lot of breaking changes!

Reply:
@antao How do you mean ? I only see one usage for it which is here:
https://github.com/bunq/sdk_csharp/blob/c71114b58d95293be99416f13e920c45f28d69c8/BunqSdk/Context/ApiContext.cs#L196

Reply:
Ow wait shit your right! I understand. Therefore this pull request needs to actually wait for `0.13.0` 👍 Because this is indeed a breaking change! Holy crap how did I miss this 😅

Reply:
@OGKevin actually the ones I changed in ApiContext.cs are all with the private modifier, so that will be alright.

Reply:
@antao Correct, but the field constants for the generated code for instance all will be used publicly, means current implementations will break!

Reply:
Lets continue the breaking change discussion in the issue please 👍